templates/erp/index.html

```diff
@@ -41,11 +41,13 @@
                 {% include "common/map.html" with map_classes="a4a-app-erp-map rounded shadow-sm" %}
             </aside>
             <div class="d-flex mb-2">
-                <h5 class="h5 inline mr-2">{% translate "Informations sur l'accessibilité" %}</h5>
-                {% if erp.checked_up_to_date_at %}
-                    {% blocktranslate trimmed with date=erp.checked_up_to_date_at|date %}
-                        - mises à jour le {{ date }}
-                    {% endblocktranslate %}
+                <h5 class="h5 inline bold mr-2">{% translate "Informations sur l'accessibilité" %}</h5>
```

Comment:
Maybe remove the h5 class as you removed the associated css style, no?

Reply:
The h5 class is still needed in this case because it triggers more styles at other places in the CSS (for example the font size)